### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by nickmorales** [virenvshah/CourseEvaluations#14]
This project aims to classify course evaluations into the following categories: "Nasty", "Constructive", and "Specific". The data is from the Cornell Course Evaluation Dataset which is an anonymized set of student's course evaluations. The team then labeled 1062 comments to be either "Nasty", "Constructive", or "Specific". This project does a great job exploring a number of natural language processing (NLP) models, including bag of words, term frequency-inverse document frequency (tf-idf),  BERT embeddings, and pre-trained embedding (Word2Vec) models. The embedding models present clear separation in the resulting PCA plots, meaning that they can extract features relevant to classifying these labels. The project then looks at unsupervised methods like latent dirichlet allocation (LDA) for summarizing comments. Furthermore, the binary classification of constructive vs specific was tested in a feed forward neural network. The group does a nice job presenting the results visually.

One minor thing that was confusing was the sentiment of comment section because it was not clear what was the method used. I thought it was strange that the "Topic Model" linear regression had a higher test accuracy than the training accuracy, but maybe I am wrong. The project does a good job discussing fairness and limitations of the models presented. It could have been useful to discuss how the inherent subjectivity and bias involved in the manual annotation of "Nasty", "Constructive", and "Specific" could have been improved.
